site.ts

```diff
@@ -189,29 +189,44 @@ function dateDiffFromToday(date: string): number {
     return diffDays
 }
 
-function createPopUp(posthog: any, config: Config, shadow: ShadowRoot, bookButtonURL: string, featureFlagName: string) {
+function createPopUp(
+    posthog: any,
+    config: Config,
+    shadow: ShadowRoot,
+    bookButtonURL: string,
+    featureFlagName: string,
+    flagInvitationTitle: string,
```

Comment:
and same below, since the flag can also not have these set

```suggestion
    flagInvitationTitle?: string,
```